app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,10 @@
             <meta-data android:name="android.support.PARENT_ACTIVITY"
                 android:value=".Activities.MainActivity" />
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="45894d9e8a6bc3b8513651d6de36159e2c836e51"
```

Comment:
Apparently, by Fabric's design, this is OK. However, it doesn't conform to best practices. So I think we can keep it as it is without any problems.